oh_staff_ui/management/commands/create_mods_records.py

```diff
@@ -7,29 +7,63 @@
 logger = logging.getLogger(__name__)
 
 
-def create_mods_records(item_id: int) -> None:
-    """Given an item_id write the related MODS record to public location"""
-    try:
-        pi = ProjectItem.objects.get(id=item_id)
-        OralHistoryMods(pi).write_mods_record()
-    except (ProjectItem.DoesNotExist) as e:
-        logger.error(e)
-        raise CommandError(f"ProjectItem with id {item_id} does not exist")
-
-
 class Command(BaseCommand):
     help = "Django management command to generate Oral History MODS records"
 
     def add_arguments(self, parser):
         parser.add_argument(
             "-i",
             "--item_id",
+            nargs="+",
```

Comment:
Since multiple items can be specified, like `-i 1 3 243`, the help should make that clear.  Or remove that capability, since exporting multiples probably should also support the counting and error handling.  I vote for removing it, but up to you.

Reply:
Noted, this was more of a nice to have but not required. Removing.